gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,20 +1,5 @@
-#
-# Copyright (c) 2011-2019 Pivotal Software Inc, All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#       https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
```

Comment:
Why is the copyright removed?

Reply:
good question, this change was applied by Gradle itself when invoking `gradle wrapper --gradle-version 5.6.4`. Do we really need it? That file is generated by Gradle...

Reply:
I prefer to keep it

Reply:
as discussed offband, keeping the removal in the PR for now